Data/Csv/Conversion.hs

```diff
@@ -798,6 +799,26 @@ instance FromField () where
     parseField _ = pure ()
     {-# INLINE parseField #-}
 
+-- | Alternative to the instance for `Only`. @since 0.5.2.0
```

Comment:
I'm afraid that won't work; the `@since` annotation needs to be a paragraph of its own.